scripts/bootstrap-profile-tools.sh

```
@@ -0,0 +1,438 @@
+	log_info "bootstrap: running: $*"
+	if ! "$@"; then
+		log_error "bootstrap: command failed: $*"
+		fail_rollback
+	fi
+}
+
+# shellcheck disable=SC2086
+[ -n "$COMPOSER_DEV" ]  && { run_or_rollback composer --working-dir="$TARGET" require --dev $COMPOSER_DEV; TOUCHED_COMPOSER=1; }
+# shellcheck disable=SC2086
+[ -n "$COMPOSER_PROD" ] && { run_or_rollback composer --working-dir="$TARGET" require $COMPOSER_PROD; TOUCHED_COMPOSER=1; }
+case "$NPM_PM" in
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this case expression.

(SC1009)